testnets/celestiatestnet2/chain.json

```diff
@@ -26,12 +24,15 @@
   },
   "codebase": {
     "git_repo": "https://github.com/celestiaorg/celestia-app",
-    "recommended_version": "v2.1.2",
+    "recommended_version": "v3.3.1-arabica",
     "compatible_versions": [
```

Comment:
This compatible_versions array doesn't match the equivalent array in the versions file.